src/client/components/JobResultPage/JobResultPage.component.js

```diff
@@ -0,0 +1,89 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import PageHeader from '../PageHeaderComponent/PageHeader';
+import JobCard from '../JobCard/JobCard';
+import ResultJobFeatures from '../ResultJobFeatures/ResultJobFeatures';
+import Button from '../Buttons/Buttons';
+import CircuitBoardBackgroundImage from '../../assets/images/circuit_board_background_image.png';
+import RobotLogo from '../../assets/images/robot_logo.png';
+import ArrowImage from '../../assets/images/arrow_next.png';
+import './JobResultPage.css';
+
+const alternateCareerArray = [
+  {
+    title: 'Full stack developer',
+    alternate: ['Front end development', 'Back end development'],
+  },
+  {
+    title: 'Data analyst',
+    alternate: [],
+  },
+  {
+    title: 'UX designer',
+    alternate: ['User interface design'],
+  },
+];
+
+const JobResultPage = ({ jobTitle }) => {
+  return (
+    <div style={{ backgroundImage: `url(${CircuitBoardBackgroundImage})` }}>
+      <PageHeader />
+      <div className="sections">
+        <div className="background-added-section">
+          <div className="career-div">
+            <div className="bigger-robot-div">
+              <img src={RobotLogo} alt="Robot logo" />
+              <p>CAREER</p>
+              <p>FOR YOU IS</p>
+            </div>
+            <JobCard jobTitle={jobTitle} />
+          </div>
+
+          <div className="alternate-career-div">
+            {jobTitle === 'Data analyst' ? (
```

Comment:
I have a small suggestion. For now, we know in advance that "Data analyst" has no alternate careers. However, it might change in the future. Maybe, we can find the alternatives for the selected job title here and check whether this array is empty or not. Then we can show empty div for empty alternatives and career options otherwise. 
Does it make sense?

Reply:
Thanks Sofiia for your comment :) I also thought that and created an empty div. Then after discussing it with the Zendesk mentor, he suggested to get rid of the empty div as we anyway have some space there. So it doesn't make any difference.